ora2/jobs/SetOra2VersionOnSandbox.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    FOLDER_NAME: some-folder
+    NOTIFY_ON_FAILURE : e-mail-contact@example.com
+*/
+
+package ora2.jobs
+
+class SetOra2VersionOnSandbox {
+
+    public static def job = { dslFactory, extraVars ->
+        def jobRoot = extraVars.get("FOLDER_NAME")
+
+        def jobName = 'SetOra2VersionOnSandbox'
+        if (jobRoot != null && jobRoot != '') {
+            jobName = jobRoot + '/SetOra2VersionOnSandbox'
```

Comment:
Oh, it might be in order to do the dynamic calculations below of the downstream job.